src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+const Person = ({ person }) => {
+  const { name, age, isMarried, partnerName } = person;
+  const maritalStatus = isMarried
+    ? `Spouse: ${partnerName}`
+    : 'I am not married';
+  const renderAge = age ? <p className="Person__age">Age: {age}</p> : null;
+
+  return (
+    <div className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {renderAge}
```

Comment:
```suggestion
      {age && <p className="Person__age">Age: {age}</p>}
```
you can use it directly